editors/vscode/client/package.json

```diff
@@ -0,0 +1,30 @@
+{
+	"name": "dada-language-server",
+	"description": "Dada Language Server",
+	"author": "Dada contributors",
+	"license": "MIT",
```

Comment:
should we make this "or apache"?

Reply:
Yes. I wrote this before https://github.com/dada-lang/dada/pull/101.